python/cugraph-dgl/cugraph_dgl/__init__.py

```diff
@@ -25,3 +26,19 @@
 import cugraph_dgl.nn
 
 from cugraph_dgl._version import __git_commit__, __version__
+
+
+def CuGraphStorage(*args, **kwargs):
+    warnings.warn(
+        "CuGraphStorage and the rest of the dask-based API are deprecated"
+        "and will be removed in release 25.08.",
+        FutureWarning,
+    )
+    return DEPRECATED__CuGraphStorage(*args, **kwargs)
+
+
+warnings.warn(
+    "cuGraph-DGL is no longer"
+    "under active development.  We strongly recommend migrating to"
+    "cuGraph-PyG."
```

Comment:
Maybe mention "removed by release 25.08" here.

Reply:
It's not being removed in 25.08, so that would be false.  It's probably being removed in 25.12.  But everything depending on the bulk sampler will be removed in 25.08 and is marked as such.

Reply:
Got it. I misread the PR description.